lib/pure/math.nim

```diff
@@ -1211,6 +1211,41 @@ func `^`*[T: SomeNumber](x: T, y: Natural): T =
         break
       x *= x
 
+func isInteger(y: SomeFloat): bool =
```

Comment:
I do not think this should be exposed in the API. Should I add an export marker here?